data.tf

```diff
@@ -0,0 +1,99 @@
+locals {
+  mgmt_network_name    = "cft-ptl-vnet"
+  mgmt_network_rg_name = "cft-ptl-network-rg"
+
+  aks_env = var.env == "sandbox" ? "sbox" : var.env
+
+  aat_cft_vnet_name           = "cft-aat-vnet"
+  aat_cft_vnet_resource_group = "cft-aat-network-rg"
+
+  cft_aks_network_name    = "cft-${local.aks_env}-vnet"
+  cft_aks_network_rg_name = "cft-${local.aks_env}-network-rg"
+
+  preview_vnet_name           = "cft-preview-vnet"
+  preview_vnet_resource_group = "cft-preview-network-rg"
+  perftest_vnet_name           = "core-infra-vnet-perftest"
+  perftest_subnet_name         = "core-infra-subnet-mgmtperftest"
+  perftest_vnet_resource_group = "core-infra-perftest"
+}
+
+provider "azurerm" {
+  alias           = "mgmt"
+  subscription_id = var.mgmt_subscription_id
+  features {}
+}
+
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+  alias                      = "aks"
+  subscription_id            = var.aks_subscription_id
+}
+
+provider "azurerm" {
+  features {}
+  alias           = "aks_preview"
+  subscription_id = "8b6ea922-0862-443e-af15-6056e1c9b9a4"
+}
+
+provider "azurerm" {
+  features {}
+  alias           = "aks_perftest_mgmt"
+  subscription_id = "7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
+}
+
+data "azurerm_subnet" "perftest_mgmt_subnet" {
```

Comment:
this shouldn't be needed its not something of ours, it _maybe_ could be something perftesters use but no harm anyway